src/test/java/TaskListTest.java

```diff
@@ -0,0 +1,45 @@
+package test.java;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.util.ArrayList;
+
+import main.java.emis.exceptions.EmisException;
+import main.java.emis.task.Task;
+import main.java.emis.TaskList;
+import main.java.emis.task.ToDo;
+
+public class TaskListTest {
+    @Test
+    public void markAsDone_exceptionThrown() throws EmisException {
```

Comment:
perhaps the naming could be more specific, marking what as done leads to to exception being thrown?

maybe a better name would be
markIndexZeroAsDone_exceptionThrown